tests/e2e/e2e.go

```diff
@@ -109,6 +115,39 @@ var _ = ginkgo.Describe("[HyperSDK Tx Workloads]", func() {
 	})
 })
 
+var _ = ginkgo.Describe("[HyperSDK Spam Workloads]", func() {
+	ginkgo.It("Spam Workload", func() {
+		if spamKey == nil || spamHelper == nil {
+			return
+		}
```

Comment:
(followup) add a TODO to remove this check and make the spamKey/spamHelper required